src/Entity/Server.php

```diff
@@ -211,7 +212,7 @@ public function configuration() {
     // Create the server config.
     $registry = $plugin->getResolverRegistry();
     $server = ServerConfig::create();
-    $server->setDebug(!!$this->get('debug'));
+    $server->setDebugFlag($this->get('debug') ? DebugFlag::INCLUDE_TRACE : DebugFlag::NONE);
```

Comment:
We should check if our debug variable with is_null() and then use DebugFlag::NONE. Otherwise use whatever integer is set in our setting. Also cast to int.